packages/xo-web/src/xo-app/home/index.js

```diff
@@ -641,7 +656,11 @@ export default class Home extends Component {
   // Optionally can take the props to be able to use it in
   // componentWillReceiveProps().
   _getFilter(props = this.props) {
-    return props.location.query.s
+    const { s } = props.location.query
+    if (s === undefined) {
+      return
+    }
+    return URL_STATE_RE.exec(s)[4]
```

Comment:
This regex refers to the sorted table URL not to the home URL.

Reply:
The home URL can contain the page number and sort info because we use `SortedTable` to diplay the backed up VMs.

Reply:
Not in the `s` var. 

https://github.com/vatesfr/xen-orchestra/blob/0f0ebca29cdb8be285e049ba2348670653ce82fd/packages/xo-web/src/xo-app/home/index.js#L690